scripts/index.js

```diff
@@ -21,62 +21,72 @@ groups.forEach(group => {
       dropdown.style.display = 'block';
       arrow.classList.add('expanded');
     }
-    document.querySelector('.welcome-container').style.display = 'flex'; // show the welcome message on click
+
+    if (window.innerWidth <= 768) {
```

Comment:
when an animal is clicked(selected) and section dropdown is closed --> welcome content is displayed(this is as expected) but the selected animal not getting deselected
![image](https://github.com/user-attachments/assets/9e616a88-a3ed-4564-9473-183cd658e8f3)
![dropdown_deselect](https://github.com/user-attachments/assets/4ca847b2-e8f4-4cc0-89fe-781d72fd85f2)